save-cloud-common/src/jvmMain/kotlin/com/saveourtool/save/utils/IdentitySourceAwareUserDetails.kt

```diff
@@ -14,11 +20,43 @@ class IdentitySourceAwareUserDetails(
     authorities: String?,
     val identitySource: String,
     val id: Long,
-) : User(
+) : org.springframework.security.core.userdetails.User(
```

Comment:
Please add an import to the class and remove the full path

Reply:
We use two different `User`  entities here, one from `save-cloud`, another from spring security, that's why we specify full path

Reply:
It's possible to add an alias import in Kotlin, e.g.:

```kotlin
import org.springframework.security.core.userdetails.User as SpringUser
```

Reply:
Fixed